docs/sources/panels-visualizations/_index.md

```diff
@@ -30,8 +30,14 @@ Panels can be dragged, dropped, and resized to rearrange them on the dashboard.
 
 Before you add a panel, ensure that you have configured a data source.
 
-- For more information about adding and managing data sources as an administrator, refer to [Data source management][].
-- For details about using specific data sources, refer to [Data sources][].
+- For details about using data sources, refer to [Data sources][].
+
+- For more information about managing data sources as an administrator, refer to [Data source management][].
+  
+  {{% admonition type="note" %}}
+  Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) and [Grafana Cloud(/docs/grafana-cloud).
+  {{% /admonition %}}
```

Comment:
You can have use and manage data sources in Grafana OSS, Enterprise, or Cloud. What is this note referring to specifically?

Reply:
Actually Managing Data Sources feature is only available in Grafana Enterprise and Cloud as there you can manage special settings, permissions, roles etc which is not available in OSS